src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MemcachedSessionHandler.php

```diff
@@ -101,7 +101,13 @@ public function write($sessionId, $data)
      */
     public function destroy($sessionId)
     {
-        return $this->memcached->delete($this->prefix.$sessionId);
+        $result = $this->memcached->delete($this->prefix.$sessionId);
+
+        if (!$result && $this->memcached->getResultCode() == \Memcached::RES_NOTFOUND) {
```

Comment:
`return $result || $this->memcached->getResultCode() == \Memcached::RES_NOTFOUND;`